src/wp-admin/theme-editor.php

```diff
@@ -165,9 +165,9 @@
 		$functions = wp_doc_link_parse( $content );
 
 		$docs_select  = '<select name="docs-list" id="docs-list">';
-		$docs_select .= '<option value="">' . esc_attr__( 'Function Name&hellip;' ) . '</option>';
+		$docs_select .= '<option value="">' . esc_html__( 'Function Name&hellip;' ) . '</option>';
```

Comment:
Is this in scope for fixing the issue? Thinking it might be better captured as a follow-up task?